tasklist/app/views/tasks/new.html.erb

```diff
@@ -0,0 +1,15 @@
+<h2>Add a new task!</h2>
+<div class="create_form">
+	<%= form_for @task, method: @task.id do |f| %> 
```

Comment:
The method should correspond to the HTTP verb which I don't think is your task ID in this case